src/components/notifications/Context/components/Static/index.js

```diff
@@ -6,14 +6,24 @@ import Notification from 'components/Notification';
 import type { Props } from '../../index';
 
 export default (props: Props) => {
+    const { selectedAccount } = props;
+    const { account } = selectedAccount;
     const { location } = props.router;
-    if (!location) return null;
-
     const notifications: Array<Notification> = [];
-    // Example:
-    // if (location.state.device) {
-    //     notifications.push(<Notification key="example" type="info" title="Static example" />);
-    // }
+
+    if (!location || !selectedAccount || !account) return null;
+
+    // Ripple minimum reserve notification
+    if (location.pathname.includes('xrp') && parseInt(account.balance, 10) < 20) {
```

Comment:
use BigNumber instead of parseInt. also "20" is not a constant, it could be much higher value regarding of transactions on this account.
I will provide those data in "account.reserved" field